recipes/unzip-http/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "unzip-http" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/unzip-http-{{ version }}.tar.gz
+  sha256: 8beaf5a4f65da904386b6ac9c49f0d35ed197b4497564d727f14bc8829ce43c0
+
+build:
+  skip: true  # [py<38]
```

Comment:
Can you please file an issue upstream so that the `scripts` tag is changed to console_scripts entry_points? That would allow it to become noarch and save CI resource.

Reply:
I can take care of it soon: https://github.com/saulpw/unzip-http/issues/2

Filed an issue in the meanwhile.

Thanks!